postgresql-17.yaml

```diff
@@ -17,6 +17,7 @@ environment:
   environment:
     # Used to fetch docker scripts
     ALPINE_VERSION: "3.21"
+    PYTHON: "python3.13"
```

Comment:
Since we can't ${{vars.py-version}} here, could you set it in the `configure` step? That does mean not no longer using the configure pipeline. See tevent.yaml as an example. (We really should add environment variables as options to that pipeline...)

Otherwise, when we do automation to try and bump python versions, we may miss this one.